config/routes.rb

```diff
@@ -60,8 +67,4 @@
   get "/cookie_policy", to: "static_pages#cookie_policy"
 
   resources :contacts, only: [:new, :create]
```

Comment:
I was thinking this could be scoped under Org - but is beyond the scope of this PR as it would require some logic changes, but being scoped there, the current org will be set for the context of the mailer. Though, I was in two minds whether this could also be used to create contacts for the non-org root i.e. from orgs who have questions about this app that comes to us devs. Not something to dwell on here. Just leaving a non-blocking thought.

Reply:
@kasugaijin One thing we might want to consider for orgs contacting devs is allow them to add tags like question or bug or feature request or important, etc. to make it easier to triage any issues they are having -- maybe not as a top priority but at some point down the line. Which might be a reason to have separate routes/controllers.

Reply:
Yes that's a great idea! @MooseCowBear